applications/GeoMechanicsApplication/custom_elements/line_interface_element.cpp

```diff
@@ -37,20 +37,25 @@ Vector CalculateDeterminantsOfJacobiansAtIntegrationPoints(const Geo::Integratio
     return result;
 }
 
-std::vector<Matrix> CalculateConstitutiveMatricesAtIntegrationPoints(
-    const std::vector<ConstitutiveLaw::Pointer>& rConstitutiveLaws, const Properties& rProperties)
+std::vector<Matrix> CalculateConstitutiveMatricesAtIntegrationPoints(const std::vector<ConstitutiveLaw::Pointer>& rConstitutiveLaws,
+                                                                     const Properties& rProperties,
+                                                                     const ProcessInfo& rProcessInfo,
+                                                                     const std::vector<Vector>& rRelativeDisplacements)
```

Comment:
As far as I know, we normally have a reference-to-const `ProcessInfo` instance as the final function parameter. I would suggest to apply that order here as well:
```suggestion
std::vector<Matrix> CalculateConstitutiveMatricesAtIntegrationPoints(const std::vector<ConstitutiveLaw::Pointer>& rConstitutiveLaws,
                                                                     const Properties& rProperties,
                                                                     const std::vector<Vector>& rRelativeDisplacements,
                                                                     const ProcessInfo& rProcessInfo)
```

Reply:
Done